roxie/topo/toposerver.cpp

```diff
@@ -331,6 +333,9 @@ int main(int argc, const char *argv[])
             queryLogMsgManager()->enterQueueingMode();
             queryLogMsgManager()->setQueueDroppingLimit(512, 32);
         }
+#else
+        queryStderrLogMsgHandler()->setMessageFields(MSGFIELD_time | MSGFIELD_milliTime | MSGFIELD_thread | MSGFIELD_prefix);
```

Comment:
if using a standard logging routine, this can be acheived by having a different "logfields" default in the components defaultYaml.